style.scss

```diff
@@ -114,6 +114,8 @@
 
 @import "bower_components/scally/core/functions/string-replace";
 
+@import "core/functions/strip-unit";
```

Comment:
Ah crap, sorry. It was at this point that I noticed th e font bug and got distracted, and forgot to fix the path :(